scipy/ndimage/filters.py

```diff
@@ -20,4 +25,8 @@ def __getattr__(name):
         raise AttributeError(
             f"scipy.ndimage.filters has no attribute {name}.")
```

Comment:
You should put the deprecation message in the attribute error as well.

Reply:
Will that not be a little confusing for a function that actually doesn't exist at all?

If not, does this look ok to you?
```suggestion
            f"scipy.ndimage.filters has no attribute {name}. "
             "Please use the `scipy.ndimage` namespace, "
             "the `scipy.ndimage.filters` namespace is deprecated.")
```

Reply:
>Will that not be a little confusing for a function that actually doesn't exist at all?

It's not the function that's deprecated, it's the module namespace. So, I don't think it's confusing at all.

I would be consistent with `signal.spline`'s message:
https://github.com/scipy/scipy/blob/e4b3e6eb372b8c1d875f2adf607630a31e2a609c/scipy/signal/spline.py#L19-L21